### ISSUE TITLE
[Crash] System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service)

### ISSUE THREAD

**Issue by Proliecan** [sn4k3/UVtools#808]
### System

```Text
UVtools v4.0.5 X64
Operative system: Linux 6.6.4-100.fc38.x86_64 #1 SMP PREEMPT_DYNAMIC Sun Dec  3 18:11:27 UTC 2023 X64
Processor: AMD Ryzen 5 4500U with Radeon Graphics
Processor cores: 6
Memory RAM: 5.21 / 7.17 GB
Runtime: fedora.38-x64
Framework: .NET 6.0.25
AvaloniaUI: 11.0.6
OpenCV: 4.8.1

Path:       /home/proliecan/.local/bin/UVtools_linux-x64_v4.0.5/
Executable: /home/proliecan/.local/bin/UVtools_linux-x64_v4.0.5/UVtools
```

### Printer and Slicer

- Printer: 
- Slicer: 

### Description of the bug

```
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service)
 ---> Tmds.DBus.Protocol.DBusException: org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service
   at Tmds.DBus.Protocol.DBusConnection.MyValueTaskSource`1.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at Tmds.DBus.Protocol.DBusConnection.CallMethodAsync(MessageBuffer message)
   at Tmds.DBus.Protocol.Connection.CallMethodAsync(MessageBuffer message)
   --- End of inner exception stack trace ---
Category: Task
File: Thug_Female.encrypted.ctb  [Version: 0] [Class: CTBEncryptedFile]

Machine date time: 12/21/2023 12:13:15AM
    UTC date time: 12/20/2023 11:13:15PM
```

### How to reproduce

Try to fix a resin trap by filling it in

**Comment by sn4k3** [comment]
Duplicate of #803
